CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+master (unreleased)
```

Comment:
The `CHANGELOG.md` is changed when there is a release, so would you mind removing this from this PR and I will then merge?